### ISSUE TITLE
Creds are stored in old format, no good for JSforce

### ISSUE THREAD

**Issue by taylordowns2000** [OpenFn/core#99]
Hoping this is short and sweet, but our good friend redux-forms seems to be throwing us for a loop! When data from the UserEdit page hits the database, they keys are transformed to `login_url` (should be `loginUrl`) and `security_token` (should be `securityToken`)!

JS force fails to connect thanks to invalid login credentials. Related to #22
```
get schema() {
    // NOTE: Temporary solution for generating forms from a schema.
    //       To be replaced when we support more than just Salesforce.
    return {
      username: { type: "text", label: "Username" },
      securityToken: { type: "text", label: "Security Token" },
      password: { type: "password", label: "Password" },
      loginUrl: { type: "text", label: "Login URL" },
    }
  }
```
```
export default reduxForm(
  {
    form: 'credential',
    fields: [
      'label', 'userId', 'id',
      'body.username', 'body.securityToken',
      'body.password', 'body.loginUrl'
    ]
  },
  mapStateToProps,
  { fetchCredential, updateCredential }
)(CredentialShow);
```

**Comment by taylordowns2000** [comment]
Hoping this is short and sweet, but our good friend redux-forms seems to be throwing us for a loop! When data from the UserEdit page hits the database, they keys are transformed to  `login_url` (should be `loginUrl`) and `security_token` (should be `securityToken`)!

JS force fails to connect thanks to invalid login credentials. Related to #22 

```  
get schema() {
    // NOTE: Temporary solution for generating forms from a schema.
    //       To be replaced when we support more than just Salesforce.
    return {
      username: { type: "text", label: "Username" },
      securityToken: { type: "text", label: "Security Token" },
      password: { type: "password", label: "Password" },
      loginUrl: { type: "text", label: "Login URL" },
    }
  }
```

```
export default reduxForm(
  {
    form: 'credential',
    fields: [
      'label', 'userId', 'id',
      'body.username', 'body.securityToken',
      'body.password', 'body.loginUrl'
    ]
  },
  mapStateToProps,
  { fetchCredential, updateCredential }
)(CredentialShow);
```

**Comment by stuartc** [comment]
Good catch, that can be avoided by removing the decamelize keys when on the
update action. I think we should store the credentials in camelCase (if we
aren't already).

**Comment by taylordowns2000** [comment]
Hah! Well, it took me some time, but I did exactly what you @stuartc suggested. Jeepers, wish I'd checked the comment on this thread :-)

Check out my TODO comment... it seems that almost all of the "Actions" are manually decamelizing stuff already, then on the client we have another decamelizeKeys function. Why both? I removed the decamelize from the Client on update and it works like a charm.
